### PR TITLE
apply proper normalization to focal loss

### PR DESCRIPTION
**Pull request by rsokl**

### CODE REVIEW

File: examples/example-detector.ipynb [hunk 1]
```diff
@@ -532,44 +536,64 @@
...
-    "        total_cls_loss, total_reg_loss = loss(class_predictions, regression_predictions, \n",
-    "                                              train_cls_targs[batch], train_reg_targs[batch])\n",
+    "        aps, ars = compute_batch_stats(\n",
+    "            class_predictions,\n",
+    "            regression_predictions,\n",
+    "            train_boxes[batch],\n",
+    "            train_labels[batch],\n",
+    "            feature_map_width=16,\n",
+    "        )\n",
+    "        total_cls_loss, total_reg_loss = old_loss(\n",
```
Comment (davidmascharka) [inline examples/example-detector.ipynb]:
```suggestion
    "        total_cls_loss, total_reg_loss = loss(\n",
```

**Review by davidmascharka — CHANGES_REQUESTED** [review]

File: examples/example-detector.ipynb (hunk 1 above)
Comment (rsokl) [reply examples/example-detector.ipynb]: ha... thank goodness I got your review 😬

**Review by davidmascharka — APPROVED** [review]
Thanks for fixing my gross formatting 😬 the days before black were grim indeed

**Comment by rsokl** [comment]
We should probably run black and isort on the whole thing in a different PR.
